### ISSUE TITLE
FEATURE: Missing Search/Find functionality in the menu

### ISSUE THREAD

**Issue by deeplook** [nwhitehead/pineapple#42]
Because it's so useful... ;-)

**Comment by norvig** [comment]
Pineapple is great; congratulations. But this is the most important missing feature to me. In a browser, command-F allows me to search; in Pineapple I've lost that and I feel I really need it.

**Comment by iagopf** [comment]
I agree. This feature is really needed for the project to be spectacular

**Comment by iagopinal** [comment]
Thumbs up for the project but I agree, this feature is critical.

**Comment by nwhitehead** [comment]
Agreed, this is a good idea. Needs a bit of thought for how to make it work properly, it might involve creating a "search results" widget somewhere.

**Comment by nwhitehead** [comment]
Looks like wxWidgets WebView has methods for doing "Find" on the page, that makes things easier.
